backend/de.metas.deliveryplanning.webui/src/main/java/de/metas/deliveryplanning/webui/process/M_Delivery_Planning_GenerateShipment.java

```diff
@@ -66,6 +70,12 @@ public Object getParameterDefaultValue(final IProcessDefaultParameter parameter)
 			final ShipmentScheduleId shipmentScheduleId = helper.getShipmentInfo(getDeliveryPlanningId()).getShipmentScheduleId();
 			return helper.getQtyToDeliverByShipmentScheduleId(shipmentScheduleId);
 		}
+		else if(parameter.getColumnName().equals(PARAM_QtyAvailable))
+		{
+			final ShipmentScheduleId shipmentScheduleId = helper.getShipmentInfo(getDeliveryPlanningId()).getShipmentScheduleId();
+			final BigDecimal qtyOnHand = helper.getQtyOnHandByShipmentScheduleId(shipmentScheduleId);
```

Comment:
Please, always use Quantity instead of BigDecimal when creating new methods if you have access to the C_UOM_ID